packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -274,12 +280,28 @@ describe('HttpHandler', () => {
         .post('/show-body')
         .set('content-type', 'application/xml')
         .send('<key>value</key>')
+        .expect(415, {
+          error: {
+            code: 'UNSUPPORTED_MEDIA_TYPE',
+            message: 'Content-type application/xml is not supported.',
+            name: 'UnsupportedMediaTypeError',
+            statusCode: 415,
+          },
+        });
+    });
+
+    it('rejects unmatched request body', () => {
+      logErrorsExcept(415);
+      return client
+        .post('/show-body')
+        .set('content-type', 'application/x-xml')
```

Comment:
`application/x-xml` is so close to `application/xml` that it's difficult to spot this difference when reading the code. Please use a distinctively different content type, e.g. `text/plain`. This will also verify that we are checking the spec to determine which types are accepted, even for types that we know how to parse.

Reply:
Switch to `text/plain`.